target/linux/x86/64/config-6.6

```diff
@@ -151,7 +151,6 @@ CONFIG_DRM_FBDEV_EMULATION=y
 CONFIG_DRM_FBDEV_OVERALLOC=100
 CONFIG_DRM_GEM_SHMEM_HELPER=y
 # CONFIG_DRM_HYPERV is not set
-CONFIG_DRM_I915=y
```

Comment:
This should be constent over all x86 targets.
If you remove it here please remove it in the other places too:
```
target/linux/x86/64/config-6.6:CONFIG_DRM_I915=y
target/linux/x86/generic/config-6.6:CONFIG_DRM_I915=y
target/linux/x86/legacy/config-6.6:CONFIG_DRM_I915=y
```

Reply:
I can do it, but I am not sure what are the generic and legacy subtargets used for, so just want to make sure that this change doesn't have any side effects.

Reply:
The generic and legacy target are used for 32 bit x86 CPUs.

I would like to keep the subtargets similar to reduce maintenance effort later on.

Reply:
OK, let me update them too

Reply:
done, check the latest PR